### PR TITLE
XW-1009 show smartbanner for app on ios

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: skins/oasis/modules/OasisController.class.php [hunk 1]
```diff
@@ -91,13 +91,24 @@ public static function JsAtBottom(){
...
 	public function executeIndex($params) {
-		global $wgOut, $wgUser, $wgTitle, $wgRequest, $wgEnableAdminDashboardExt, $wgOasisThemeSettings;
+		global $wgOut, $wgUser, $wgTitle, $wgRequest, $wgEnableAdminDashboardExt, $wgOasisThemeSettings,
+		$wgWikiaMobileSmartBannerConfig;
 
 		wfProfileIn(__METHOD__);
 
 		//Add Smart banner for My Wikia App
 		//See: https://wikia-inc.atlassian.net/browse/MOB-167
-		$wgOut->addHeadItem('My Wikia Smart Banner', '<meta name="apple-itunes-app" content="app-id=623705389">');
```
Comment (vforge) [inline skins/oasis/modules/OasisController.class.php:100]: Don't we want to leave this as a backup/default option?

Comment (bognix) [inline skins/oasis/modules/OasisController.class.php:100]: @alistra @aniaMu - do we want to leave the My Wikia app as the default option?

Comment (alistra) [inline skins/oasis/modules/OasisController.class.php:100]: Yes, please provide MyWikia as the default if no config is set in WikiFactory (only for iPad, as MyWikia supports iPad only)

**Comment by vforge** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by hakubo** [comment]
> LGTM

**Pull request merged**